python/pyarrow/tests/util.py

```diff
@@ -22,7 +22,10 @@
 import contextlib
 import decimal
 import gc
-import numpy as np
+try:
+    import numpy as np
+except ImportError:
+    pass
```

Comment:
Is this import actually used?

Reply:
I've done some cleanup here so we don't have to import here